zulipterminal/config/themes.py

```diff
@@ -12,24 +12,24 @@
     'content',
```

Comment:
Do we not use the `content` style anywhere? I grepped the code base with 'content' but couldn't find anything.

Reply:
Right; I was mainly working on renames and prefixes at this point, but I just added a commit for this. This style has been present since at least 2018, as this tidying has been a long time coming.

Reply:
I added this as an extra commit; this style has existed for a long time, so it likely just didn't get removed after being introduced.